__tests__/task.test.js

```diff
@@ -1,38 +1,32 @@
-import { internet } from 'faker';
 import {
-  clearDatabaseState, getAuthenticatedUser, launchApp, shutdownApp,
+  create, database, getAuthenticatedUser, launchApp, shutdownApp,
 } from './helpers';
 
 describe('Task', () => {
   let app;
   let cookies;
+  let db;
   let currentUser;
   let existingStatus;
   let existingLabel;
   let existingUser;
 
   beforeAll(async () => {
     app = await launchApp();
+    db = database(app);
   });
 
   afterAll(async () => {
     await shutdownApp(app);
   });
 
   beforeEach(async () => {
-    await clearDatabaseState(app);
+    await db.clear();
     ({ user: currentUser, cookies } = await getAuthenticatedUser(app));
-    existingStatus = await app.objection.models.status.query().insert({
-      name: 'test status',
-    });
-    existingLabel = await app.objection.models.label.query().insert({
-      name: 'test label',
-    });
-    existingUser = await app.objection.models.user.query().insert({
-      firstName: 'foo',
-      lastName: 'bar',
-      email: internet.email(),
-      password: 'test',
+    existingStatus = await db.insert.status(create.status());
+    existingLabel = await db.insert.label(create.label());
+    existingUser = await db.insert.user({
+      ...create.user(),
       labelIds: existingLabel.id,
       statusId: existingStatus.id,
```

Comment:
why user has labels and statuses?

Reply:
also if there should be task, please consider to move relational logic to the builders not to the test